susemanager-utils/testing/docker/master/uyuni-master-nodejs/add_packages.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 # Packages for Javascript tests
-zypper --non-interactive in nodejs8 npm8
+zypper --non-interactive in nodejs10 npm8
```

Comment:
Shouldn't we require npm10?

Reply:
well neither npm8 or npm10 exists ahahah I will delete that. Npm is already installed by default with node.js

Reply:
Done.

Btw should I create an issue to update this Urls: https://github.com/uyuni-project/uyuni/blob/102ac09eb0a2c72843bd8a93bfb693cc70392f69/susemanager-utils/testing/docker/master/uyuni-master-root/add_repositories.sh#L7 and rebuild all the docker images? or are you already aware of that?

Reply:
Warning: are you sure it's installed by default? I think it is just a recommend!

Reply:
Mmmmmm, we could have a problem about this, I am afraid.

It seems Leap 15.0 does NOT have nodejs10, but only nodejs8!